### PR TITLE
test, probes: Replace Fedora with Alpine

### PR DESCRIPTION
**Pull request by qinqon**
Signed-off-by: Enrique Llorente <ellorent@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
Depends on https://github.com/kubevirt/kubevirtci/pull/801

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by iholder-redhat** [comment]
Looks like a blessed change unless for some reason there's a special need for fedora.
Thanks!
/lgtm

**Comment by qinqon** [comment]
> Looks like a blessed change unless for some reason there's a special need for fedora. Thanks! /lgtm

The idea is to do some replacements at test and compare with fedora at CI to see if it's really faster.

**Comment by iholder-redhat** [comment]
> > Looks like a blessed change unless for some reason there's a special need for fedora. Thanks! /lgtm
> 
> The idea is to do some replacements at test and compare with fedora at CI to see if it's really faster.

Good idea, sounds great :+1:

**Comment by qinqon** [comment]
Add the rest of network tests except SRIOV, let's check it out the times now.

File: WORKSPACE [hunk 1]
```diff
@@ -337,10 +337,10 @@ container_pull(
 
 container_pull(
     name = "alpine_with_test_tooling",
-    digest = "sha256:48181a73ecc9c076ad30d8e31202e46a6c4ff4aff7a17b098b8864f2b8f099df",
+    digest = "sha256:2409034f1990bf72b9fb8a593cab1f05def148fb2b4c9bc52b4b8a5f3cff4fd5",
     registry = "quay.io",
-    repository = "kubevirtci/alpine-with-test-tooling-container-disk",
-    tag = "2204130933-caa3837",
+    repository = "ellorent/alpine-cloud-init",
```
[the remark this answers is not in the document]
Comment (qinqon) [reply WORKSPACE:342]: Yep, we need proper ICMP ping tool so iptools is needed and also dhclient so we use the same logic as fedora to test IPv6.

**Pull request closed without merging**